src/compiler/declarationEmitter.ts

```diff
@@ -834,6 +834,7 @@ namespace ts {
         }
 
         function writeModuleDeclaration(node: ModuleDeclaration) {
+            if (!node.body) { throw new Error("TODO"); } //TODO

```

Comment:
should be valid. you want to write it the same way you saw it. i.e. short hand declaration.